packages/devtools_app/lib/src/network/flutter/http_timing_tab.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../http_request_data.dart';
+
+/// A [Widget] which displays timing information for an HTTP request.
+class HttpRequestTimingTab extends StatelessWidget {
+  const HttpRequestTimingTab(this.data);
+
+  @override
+  Widget build(BuildContext context) {
+    final events = <Widget>[];
+    for (final instant in data.instantEvents) {
+      final timeDiffMillis = instant.timeDiffMs;
+      events.add(_buildTile(
+          instant.name, [_buildRow('Duration', '$timeDiffMillis ms')]));
+    }
+    events.add(
+        _buildTile('Total', [_buildRow('Duration', '${data.durationMs} ms')]));
+
+    return Padding(
+      padding: const EdgeInsets.only(left: 14.0, top: 18.0),
```

Comment:
make 14 and 18 constants as they are used a lot of places. that way if we want to tweak the padding of the ui consistently it will be easy and the constant names can document the intent of the values.